java/ql/src/Security/CWE/CWE-1204/StaticInitializationVector.qhelp

```diff
@@ -3,11 +3,10 @@
 
 <overview>
 <p>
-A cipher needs an initialization vector (IV) when it is used in certain modes
-such as CBC or GCM. Under the same secret key, IVs should be unique and ideally unpredictable.
-Given a secret key, if the same IV is used for encryption, the same plaintexts result in the same ciphertexts.
-This lets an attacker learn if the same data pieces are transferred or stored,
-or this can help the attacker run a dictionary attack.
+When a cipher is used in certain modes such as CBC or GCM, it requires an initialization vector (IV).
+Under the same secret key, IVs should be unique and ideally unpredictable.
```

Comment:
Making sure `unpredictable` is the word you meant to use here.

Reply:
Also, on line 21, let's use a sentence to break two clauses:
`The following example initializes a cipher with a static IV, which is unsafe:`